Tests.Template/changelist.md

```diff
@@ -21,6 +21,9 @@ When the templates are used to create or update tests in a DSC Resource the vers
 * First release including version number.
 
 ## unit_template.ps1
+### Unreleased ###
+* Update Pester syntax to v4
+
```

Comment:
These test templates don't get released. They just stay here in this repo for other people to copy.
So can you increment the version number in the comment in the template and put the same version number here in place of 'Unreleased' as well?